File: c++_courses/primeNumber.cpp
#include <iostream>






int main(int argc, char const *argv[])
{

	bool is_prime=true;
	int n;

	std::cout<<"Enter a number to be checked: ";
	std::cin>>n;

	
	std::cout<<"Your Number is: "<<n<<"\n";

    if (n==0||n==1) {
        is_prime=false;
    }



	for (int i = 2; i < n; ++i)
	{
		/* code */
		if (n%2==0)
		{
			/* code */
			is_prime=false;
			break;	
		}

	}
	if (is_prime)
	{
		/* code */
		std::cout<<"It is a prime number"<<std::endl;
	}
	else{
		std::cout<<"It is not a prime number"<<std::endl;
	}
	return 0;
}

File: jajava/primenumber.java
class PrimeExample{
    public static void main(String[] args) {
        int i,m=0,flag=0;
        int n=3;//it is the number to be check
        boolean is_prime=true;

        if (n==0||n==1) {
            is_prime=false;
        }



        for (i=2;i<n;n++) {
            if (n%2==0) {
                is_prime=false;
                break;             
            }
        }
        if (is_prime) {
        System.out.println("is a prime number");          
        } else {
        System.out.println("is not a prime number");
            
        }



    }
}